go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/hashicorp/errwrap v1.0.0
 	github.com/hashicorp/go-cleanhttp v0.5.1
-	github.com/hashicorp/go-getter v1.4.0
+	github.com/hashicorp/go-getter v1.4.1
```

Comment:
Was there a strong reason for bumping this? The reason I'm asking is because this seems to require Go 1.13 as described in https://github.com/hashicorp/go-getter/pull/233

Reply:
No, I think v1.4.0 will work fine. Will revert this.